src/models/InstabugConfig.ts

```diff
@@ -1,6 +1,8 @@
 import type { invocationEvent } from '../utils/ArgsRegistry';
+import type { LogLevel } from '../utils/Enums';
 
 export interface InstabugConfig {
   token: string;
   invocationEvents: invocationEvent[];
+  debugLogsLevel?: LogLevel;
```

Comment:
**suggestion:** We should add the doc comments to these properties, instead of `Instabug.init`. Let's apply to all properties.

```suggestion
  /**
   * An optional LogLevel to indicate the verbosity of SDK logs. Default is Error.
   */
  debugLogsLevel?: LogLevel;
```